project/Dependencies.scala

```diff
@@ -186,4 +189,13 @@ object Dependencies {
     Seq(specs2core, typeSafeConfig)
   }
 
+  def databaseDependencies: Seq[ModuleID] = {
+    lazy val scalaTest  = "org.scalatest"   %% "scalatest"  % Versions.scalatest  % "test"
```

Comment:
```suggestion
    lazy val scalaTest  = "org.scalatest"   %% "scalatest"  % Versions.scalatest  % Test
```

Wouldn't this also work? Just saw it also in this file